website/templates/emails/preprint_confirmation_branded.txt.mako

```diff
@@ -0,0 +1,37 @@
+<%!
+    from website import settings
+%>
+
+Hello ${user.fullname},
+
+Congratulations on sharing your ${preprint.provider.preprint_word}  "${node.title}" on ${preprint.provider.name}, powered by OSF Preprints: ${preprint.absolute_url}
+
+Now that you've shared your ${preprint.provider.preprint_word}, take advantage of more OSF features:
+
+*Upload supplemental, materials, data, and code to the OSF project associated with your ${preprint.provider.preprint_word}: ${node.absolute_url}
+Learn how: http://help.osf.io/m/preprints/l/685323-add-supplemental-files-to-a-preprint
+
+*Preregister your next study and become eligible for a $1000 prize: osf.io/prereg
+
+*Track your impact with ${preprint.provider.preprint_word} downloads
+
+
+
+You will ${'not receive ' if all_global_subscriptions_none else 'be automatically subscribed to '}notification emails for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}
+
+Sincerely,
+
+Your ${preprint.provider.name} and OSF teams
+
+Want more information? Visit https://osf.io/preprints/${preprint.provider.name.lower()} to learn about ${preprint.provider.name} or https://osf.io/ to learn about the Open Science Framework, or https://cos.io/ for information about its supporting organization, the Center for Open Science.
```

Comment:
This should probably use preprint.provider.domain if it redirects instead of building the `preprints/provider` route. (though this is probably a product question as well, not sure if custom domains existed when the ticket was created)

Might be worth adding a provider `url` method similar to preprints `url` method [seen here](https://github.com/CenterForOpenScience/osf.io/blob/develop/osf/models/preprint_service.py#L86).